src/wallet/wallet.cpp

```diff
@@ -1768,6 +1770,15 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
             // scan succeeded, record block as most recent successfully scanned
             result.last_scanned_block = block_hash;
             result.last_scanned_height = block_height;
+
+            if (save_progress && next_interval) {
+                CBlockLocator loc= m_chain->getActiveChainLocator(block_hash);
```

Comment:
nit: missing space before "="

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25036/commits/a89ddfbe22b6db5beda678c9493e08fec6144122. Thanks